### ISSUE TITLE
CI and PR test builds are failing on wait conditions

### ISSUE THREAD

**Issue by geerlingguy** [ansible-collections/community.kubernetes#104]
##### SUMMARY

I thought it would just be a one-time issue, but it seems like the past few runs—for PRs as well as scheduled CI jobs—have all been failing on this task:

```
    TASK [Check that deployment wait worked] ***************************************
    task path: /home/runner/work/community.kubernetes/community.kubernetes/ansible_collections/community/kubernetes/molecule/default/tasks/waiter.yml:241
fatal: [localhost]: FAILED! => {
    "assertion": "updated_deploy_pods.resources[0].spec.containers[0].image.endswith(\":2\")",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}
    Read vars_file 'vars/main.yml'
```

This problem started appearing after merging https://github.com/ansible-collections/community.kubernetes/pull/35 — though I'm not 100% sure what the issue is here.

Failing build example: https://github.com/ansible-collections/community.kubernetes/runs/695417261?check_suite_focus=true#step:8:25496

cc @willthames, @fabianvf 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

wait conditions

##### ANSIBLE VERSION

N/A

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

GitHub Actions CI

##### STEPS TO REPRODUCE

Run CI build.

**Comment by willthames** [comment]
Hopefully #100 should fix the last of these but I will definitely investigate any others.

**Comment by geerlingguy** [comment]
I believe this has been adequately fixed by #100, and I haven't seen any related failures in CI since.
